apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -55,10 +55,10 @@ class PageTemplateModal extends Component {
 		this.props.insertTemplate( title, previewBlocks );
 	};
 
-	selectTemplate = () =>
+	handleConfirmation = () =>
```

Comment:
I don't get this renaming tbh. What the function does is select the template once the user clicks on the select template button. There is not any a confirmation to handle.

Reply:
True, we now have and extra button that user needs to click and I view that as a "confirmation". By using a different word, I'm trying to help the confusion between words `select`, `set` and `insert` which honestly sound like the exactly same thing and unless you read their source code, there is no way to guess what they do.

Reply:
my last two coins: `onSelectTemplateHandler`. Feel free to ignore me.

Reply:
I would really like to avoid `select` here. We have already used this word for a different interaction - when user clicks the template (`onTemplateSelect`). If we use "select" again in the same file but with so different meaning, I think that's a source of confusion.

Reply:
@marekhrabe How about swapping `onTemplateSelect` with `onPreviewClick` and keeping `selectTemplate`?